### PR TITLE
Feature/formsubmit

### PR DESCRIPTION
**Pull request by marcos-de-barros-vioque**
Hi!
In this PR I've added the on submit handler to the addspendingform and created the spendings page. 
After having submitted the form, the user is redirected to the spendings page where the form input is displayed.
From the spendings page, the user can return to the add spendings page by clicking on the "Add New Spending" button.
Cheers

***UPDATE: This PR has been updated according to QA feedback.***

### CODE REVIEW

**Review by stefankorder — APPROVED** [review]
LGTM!

File: components/AddSpendingForm.js [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
+`;
+
 const StyledAddSpendingForm = styled.form`
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  width: 30%;
+  margin-top: 10px;
+  margin-bottom: 100px;
```
Comment (stefankorder) [inline components/AddSpendingForm.js:67]: Better use relative units at this point.

File: components/AddSpendingForm.js [hunk 2]
```diff
@@ -38,13 +82,19 @@ const StyledFormInput = styled.input`
   font-size: 15px;
```
Comment (stefankorder) [inline components/AddSpendingForm.js:82]: Better use relative units at this point.

File: components/AddSpendingForm.js [hunk 3]
```diff
@@ -38,13 +82,19 @@ const StyledFormInput = styled.input`
...
+  border-radius: 5px;
+  padding: 10px;
+  font-size: 15px;
+`;
+
 const StyledFormButton = styled.button`
   align-self: center;
   text-align: center;
-  margin: 20px 0 20px 0;
+  margin: 25px 0 20px 0;
```
Comment (stefankorder) [inline components/AddSpendingForm.js:94]: Better use relative units at this point.

File: pages/index.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ const StyledBody = styled.div`
   justify-content: center;
   align-items: center;
   width: 100%;
-  margin-top: 150px;
+  margin-top: 200px;
```
Comment (stefankorder) [inline pages/index.js:40]: Better use relative units at this point.

**Review by stefankorder — CHANGES_REQUESTED** [review]
LGTM! Please take a look at my comments.

**Review by stefankorder — APPROVED** [review]
LGTM!

File: components/Header.js [hunk 5]
```diff
@@ -23,10 +23,11 @@ const StyledHeaderContainer = styled.div`
...
   position: fixed;
   width: 100%;
+  background-color: #549b8c;
   top: 0;
 `;
 
 const StyledLogo = styled(Link)`
   text-decoration: none;
-  margin: 30px 0px 0px 30px;
+  margin: 10px 0px 10px 30px;
```
[the remark this answers is not in the document]
Comment (marcos-de-barros-vioque) [reply components/Header.js]: Thanks @stefankorder ! Should I repost this PR in the code-reviews channel, or rather move to QA?

**Pull request merged**